src/app/api/api_v1/endpoints/detections.py

```diff
@@ -112,10 +112,33 @@ async def fetch_detections(
     if UserRole.ADMIN in token_payload.scopes:
         return [elt for elt in await detections.fetch_all()]
 
-    cameras_list = await cameras.fetch_all(("organization_id", token_payload.organization_id))
+    cameras_list = await cameras.fetch_all(filter_pair=("organization_id", token_payload.organization_id))
     camera_ids = [camera.id for camera in cameras_list]
 
-    return await detections.get_in(camera_ids, "camera_id")
+    return await detections.fetch_all(in_pair=("camera_id", camera_ids))
+
+
+@router.get("/unlabeled/fromdate", status_code=status.HTTP_200_OK, summary="Fetch all the unlabeled detections")
+async def fetch_unlabeled_detections(
+    from_date: datetime = Query(),
+    detections: DetectionCRUD = Depends(get_detection_crud),
+    cameras: CameraCRUD = Depends(get_camera_crud),
+    token_payload: TokenPayload = Security(get_jwt, scopes=[UserRole.ADMIN, UserRole.AGENT, UserRole.USER]),
+) -> List[Detection]:
+    telemetry_client.capture(token_payload.sub, event="unlabeled-fetch")
+
+    all_unck_detections = await detections.fetch_all(filter_pair=("is_wildfire", None))
+    all_unck_detections = [detection for detection in all_unck_detections if detection.created_at >= from_date]
+
+    if UserRole.ADMIN in token_payload.scopes:
+        return all_unck_detections
+
+    cameras_list = await cameras.fetch_all(filter_pair=("organization_id", token_payload.organization_id))
+    camera_ids = [camera.id for camera in cameras_list]
+    all_unck_detections = await detections.fetch_all(
+        filter_pair=("is_wildfire", None), in_pair=("camera_id", camera_ids)
+    )
```

Comment:
if you're not an admin, fetch_all will be called twice while we only need it once